src/app.py

```diff
@@ -371,18 +433,23 @@ def load_file(filename: str) -> dict:
     st.divider()
 
     st.subheader(":blue-background[Dependabot Alerts]")
-    st.write("Alerts open for more than 5 days (Critical), 15 days (High), 60 days (Medium), 90 days (Low).")
-
     # Rename the columns of the DataFrame
     df_dependabot.columns = [
         "Repository Name",
         "Type",
+        "Created At",
         "Dependency",
         "Advisory",
         "Severity",
         "Days Open",
         "Link",
     ]
+    max_days_open = int(df_dependabot["Days Open"].max()) if not df_dependabot["Days Open"].empty else 0
```

Comment:
for readability purposes, it might be better to flip this. 

0 if empty else value

instead of

value if not empty else 0

I know it seems like a small change, but the readability is better (I think).